class ValidateError(Exception):
    pass

class AuthorizationError(Exception):
    pass